articles/becoming-a-flutter-developer/index.md

```diff
@@ -0,0 +1,179 @@
+
+4. **DevTools**
+
+It is a tooling web suite for Flutter and Dart developers comprising a layout inspection tools, performance tools and memory tools. It contains all the debugging tools that you need to be an efficient and effective Flutter developer. 
+
+5. **Design Tools**
+
+These are tools used to craft and visualize ideas. Design is an important stage of app development. It ensures for a smooth software development process. Some software development teams include UI/UX designers, especially for building commercial applications while others lack. Building personal projects will also need knowledge of design tools.
+
+Here are some of the most popular design tools:
+- Sketch.
+- Adobe XD.
+- [Figma.](https://www.figma.com/) 
+- [Flutter Studio.](https://flutterstudio.app/)
+
+After designing, you can use [Supernova](https://www.supernova.io/) to convert the design into code by exporting it. You will, however, need to refactor the exported code to break it into readable code.
+
+### Flutter Developer Skills
+
+1. **Dart Programming Language**
+
+`Dart` is a client-based programming language that is scalable and can write simple scripts or full-featured applications. It is an object-oriented and dynamic language. Google developed dart. It is the primary language used in Flutter.
+
+2. **Knowledge on Agile Methodology and Waterfall for Software Development**
+
+A software development method is dividing the software development process into different phases. It enhances product design and resource utilization ***for example*** money and time. 
+
+The most commonly used methodologies are `Waterfall Methodology` and `Agile Methodology`. A Flutter developer needs to familiarize themselves with the software methodologies to work with ease in a software development team.
+
+3. **Prior knowledge on a Native Language**
+
+The native languages for mobile application development include:
+
+1. `Swift` for iOS applications.
+
+2. `Java` and `Kotlin` for Android applications.
+
+Understanding these languages will make it easier to become a flutter developer. Learning Dart with the knowledge of a native language makes it easier.
+
+
+### Mistakes to Avoid When Learning Flutter Development
+
+- **Learn native languages, not just flutter and dart.**
+
+Most software developers will argue that prior knowledge in a native language is unnecessary, but it is a skill requirement on most job descriptions. Flutter development is still fairly new, and employers must have proof that you know about application development. The only way to show this is by having experience with native languages were the only ones used to develop most commercial and large-scale applications.
+
+Knowing a native language will give you a groundbreaking compared to a person who only knows flutter and dart. It will display a variety of expertise and knowledge. A grasp on native languages will make it easier to move flutter development.
```

Comment:
In this sentence
`Knowing a native language will give you a groundbreaking compared to a person who only knows flutter and dart`
`groundbreaking` can simply be replaced with `advantage`. Some users may not be familiar with that term.